var/spack/repos/builtin/packages/py-minrpc/package.py.save

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
```

Comment:
It looks like this file accidentally slipped into your PR

Reply:
Sorry!